projects/arlas-toolkit/src/lib/components/histogram-tooltip-overlay/histogram-tooltip-overlay.component.ts

```diff
@@ -21,14 +21,51 @@
 import { Component, Inject } from '@angular/core';
 import { ArlasOverlayRef, HISTOGRAM_TOOLTIP_DATA } from '../../tools/utils';
 import { HistogramTooltip } from 'arlas-web-components';
+import moment from 'moment';
 
 @Component({
   selector: 'arlas-histogram-tooltip-overlay',
   templateUrl: './histogram-tooltip-overlay.component.html',
   styleUrls: ['./histogram-tooltip-overlay.component.css']
 })
 export class HistogramTooltipOverlayComponent {
+  public interval: {
+    start: Date | number;
+    end: Date | number;
+  };
 
-  public constructor(public overlayRef: ArlasOverlayRef, @Inject(HISTOGRAM_TOOLTIP_DATA) public tooltip: HistogramTooltip) {}
+  public constructor(public overlayRef: ArlasOverlayRef, @Inject(HISTOGRAM_TOOLTIP_DATA) public tooltip: HistogramTooltip) {
+    this.calculateDate();
+  }
+
+
+  public calculateDate(){
+    if(this.tooltip.dataType === 'time'){
+      const start =  new Date(this.tooltip.xValue);
+      const end = moment(start, 'DD MMMM YYYY', true).add(this.tooltip.xRange.value, 'd').toDate();
```

Comment:
I fear that they might have some utc/non-utc incoherence here; especially if the start date in in utc.

Reply:
what would you do ?